### PR TITLE
[WIP] LWRP for adding Sensu Datacenters to Uchiwa

### PR DESCRIPTION
**Pull request by amdprophet**

### CODE REVIEW

**Comment by palourde** [comment]
jdb9oxc

File: providers/sensu_datacenter.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  datacenter[:host] = new_resource.host
+  datacenter[:port] = new_resource.port
+  datacenter[:ssl] = new_resource.ssl
+  datacenter[:insecure] = new_resource.insecure
+  datacenter[:path] = new_resource.path unless new_resource.path.nil?
+  datacenter[:timeout] = new_resource.timeout
+  datacenter[:user] = new_resource.user unless new_resource.user.nil?
+  datacenter[:pass] = new_resource.pass unless new_resource.pass.nil?
+
+  node.default["uchiwa"]["datacenters"] << datacenter
```
Comment (portertech) [inline providers/sensu_datacenter.rb]: Shouldn't this first check for the existence of a datacenter with the same name, then compare hash objects before adding to the node attribute array? Can then notify resource updated if it's added.

Comment (amdprophet) [inline providers/sensu_datacenter.rb]: I believe it's possible to have two data centers with the same name but yes I can do that.

**Comment by majormoses** [comment]
@amdprophet status?

**Comment by amdprophet** [comment]
This can be closed unless someone else wants to try to complete the work.

**Comment by majormoses** [comment]
closing per: https://github.com/sensu/uchiwa-chef/pull/41#issuecomment-329016745 as much as I would like to see this I have no need right now so I can't really justify the time to pick this up myself.

**Pull request closed without merging**